libs/eppi_importer/import_from_eppi.py

```diff
@@ -0,0 +1,48 @@
+"""Import references from a EPPI export file and write them to a .jsonl file."""
+
+import argparse
+import base64
+import hashlib
+import json
+from pathlib import Path
+
+from destiny_sdk.parsers.eppi_parser import EPPIParser
+
+
+def main() -> None:
+    """Import references from a EPPI export and write them to a .jsonl file."""
+    arg_parser = argparse.ArgumentParser(
+        description="Import references from a EPPI export file."
+    )
+    arg_parser.add_argument(
+        "--input", "-i", type=str, required=True, help="Input EPPI export filename"
+    )
+    arg_parser.add_argument(
+        "--output", "-o", type=str, required=True, help="Output .jsonl filename"
+    )
+    arg_parser.add_argument(
+        "--tags",
+        nargs="+",
+        default=[],
+        help="A list of tags to add as annotation enhancements.",
+    )
+    args = arg_parser.parse_args()
+
+    input_path = Path(args.input)
+
+    with input_path.open("rb") as f:
+        file_bytes = f.read()
+        checksum = base64.b64encode(hashlib.md5(file_bytes).digest()).decode("ascii")  # noqa: S324
```

Comment:
MD5 was chosen for convenience to match Content-MD5 shown in Azure container file metadata.